src/content/reference/react-dom/client/index.md

```diff
@@ -1,22 +1,23 @@
 ---
-title: Client React DOM APIs
+title: API Klien React DOM
 ---
 
 <Intro>
 
-The `react-dom/client` APIs let you render React components on the client (in the browser). These APIs are typically used at the top level of your app to initialize your React tree. A [framework](/learn/start-a-new-react-project#production-grade-react-frameworks) may call them for you. Most of your components don't need to import or use them.
+
```

Comment:
```suggestion
```

Please remove this unnecessary additional empty line. 😁

Reply:
Thanks for noticing, updated @zainfathoni